### DISCUSSION TITLE
Egress provider for s3

### DISCUSSION THREAD

**Discussion by Egyptmaster** [dotnet/dotnet-monitor#2009] [Ideas]
Hi there, I worked on a egress provider for s3 and I would ask to create a pull-request to bring it to the code base. Currently I cannot push my branch as I received a 403. So can you help or let me know how to do so?

**Comment by birojnayak** [comment]
@Egyptmaster I don't think there is any sort of restrictions.. you should be able to create your own fork and push from there.... I was going to implement this, will hold for your contribution..

**Reply by Egyptmaster** [reply]
I created the pull-request accordingly. Thanks for the hint.

**Comment by jander-msft** [comment]
S3 egress was contributed in #2016
